packages/cli/src/index.ts

```diff
@@ -187,7 +187,7 @@ export async function graphqlMesh() {
               return m.default || m;
             });
 
-          const baseDirRequire = createRequire(join(baseDir, 'mesh.config.js'));
+          const baseDirRequire = createRequire(baseDir);
```

Comment:
I think it needs to be a filename according to NodeJS documentation;
https://nodejs.org/api/module.html#module_module_createrequire_filename